portfolio/src/main/webapp/style.css

```diff
@@ -117,3 +122,63 @@ html {
     flex-flow: row nowrap;
     justify-content: center;
 }
+
+.photography{
+    align-items: center;
+    background-color: #FBFBFB ;
+    padding: 0 700px 50px 700px;
+    position: relative;
+}
+
+.masonry{
+    column-gap: 40px;
+    display: grid;    
+    grid-template-columns: repeat(18,1fr);
+    grid-template-rows: repeat(9,1fr);
+    row-gap: 40px;
+}
+
+.grid-pic{
+    display: block;
+    border-radius: 5px;
+    height:100%;
+    margin: 0;
+    object-fit: cover;
+    width: 100%;
+}
+
+.cell-1{
+    grid-column: 1 / 7;
+    grid-row: 1 / 3;
+    margin: 0;
+}
```

Comment:
I feel like having to individually define properties for each image kind of defeats the purpose of using CSS Grid, though I suppose it works well enough here.

Other ideas for a masonry style site: https://css-tricks.com/piecing-together-approaches-for-a-css-masonry-layout/

Reply:
I was looking for something like this, the individual properties is definitely not good for the long run. Thanks for the heads-up, the library I was trying to use ended up being tricky so I went for a quick fix just to get something up for the day.